test/util/cert_fixtures.go

```diff
@@ -10,19 +10,21 @@ import (
 )
 
 var (
-	svidPath    = path.Join(ProjectRoot(), "test/fixture/certs/svid.pem")
-	svidKeyPath = path.Join(ProjectRoot(), "test/fixture/certs/svid_key.pem")
-	caPath      = path.Join(ProjectRoot(), "test/fixture/certs/ca.pem")
-	caKeyPath   = path.Join(ProjectRoot(), "test/fixture/certs/ca_key.pem")
-	bundlePath  = path.Join(ProjectRoot(), "test/fixture/certs/bundle.der")
+	svidPath     = path.Join(ProjectRoot(), "test/fixture/certs/svid.pem")
+	svidKeyPath  = path.Join(ProjectRoot(), "test/fixture/certs/svid_key.pem")
+	caPath       = path.Join(ProjectRoot(), "test/fixture/certs/ca.pem")
+	caKeyPath    = path.Join(ProjectRoot(), "test/fixture/certs/ca_key.pem")
+	bundlePath   = path.Join(ProjectRoot(), "test/fixture/certs/bundle.der")
+	blogCsrPath  = path.Join(ProjectRoot(), "test/fixture/certs/blog_csr.pem")
+	blogSvidPath = path.Join(ProjectRoot(), "test/fixture/certs/blog_cert.pem")
```

Comment:
What's the difference between blogSvid and the SVID above? Looks like the functions that use these aren't being called currently - can they be removed?

Reply:
I can't tell the difference between them, they were used by the old manager's tests...I think it's fine to remove them, so I will do it.